index.js

```diff
@@ -7,11 +7,4 @@ module.exports = fp => new Promise(resolve => {
 	});
 });
 
-module.exports.sync = fp => {
-	try {
-		fs.accessSync(fp);
-		return true;
-	} catch (err) {
-		return false;
-	}
-};
+module.exports.sync = fs.existsSync;
```

Comment:
I'm not interested in changing the behavior at this point. It would also make it inconsistent with the async version. Documentation change is enough.